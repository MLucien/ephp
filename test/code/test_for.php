<html>
<head>
    <title>My Web Example</title>
</head>
<body>
    <table>
    <? for ($i=0; $i<2; $i++) { ?>
    <tr>
        <td><?=$i?></td>
    </tr>
    <? } ?>
</body>
</html>

