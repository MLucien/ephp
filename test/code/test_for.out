<html>
<head>
    <title>My Web Example</title>
</head>
<body>
    <table>
        <tr>
        <td>0</td>
    </tr>
        <tr>
        <td>1</td>
    </tr>
        </table>
</body>
</html>

